bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -420,6 +426,21 @@ synchronized void logErrorAndReattemptRead(int bookieIndex, BookieSocketAddress
             }
         }
 
+        @Override
+        boolean complete(int bookieIndex, BookieSocketAddress host, ByteBuf buffer) {
+            boolean completed = super.complete(bookieIndex, host, buffer);
+            if (completed && lh.bk.getConf().getEnsemblePlacementPolicySlowBookies()) {
```

Comment:
i don't think we need to check if this is enabled here. if disabled, the placement policy should just ignore the extra info.